pyntcloud/io/ply.py

```diff
@@ -242,7 +248,7 @@ def describe_element(name, df):
     -------
     element: list[str]
     """
-    property_formats = {'f': 'float', 'u': 'uchar', 'i': 'int'}
+    property_formats = {'f': 'float', 'u': 'uchar', 'i': 'int', 'b': 'bool'}
```

Comment:
Could we add a test for `write_ply` with bool?

Reply:
https://github.com/daavoo/pyntcloud/pull/321/files#diff-021377fee185787af295603466e9f1f3fec039246017a515213eb7c2f2115888